ts/analysis-component.ts

```diff
@@ -59,6 +59,7 @@ export class AnalysisComponent {
         let renderers = $.extend($.pivotUtilities.renderers,
             { 'File export': (new FileExportRenderer()).render });
         let pivotData = this.analysisService.getFlatTable(searchResults, this.variables.map(x => x.name), metadataKeys);
+        let defaultVariable = this.variables.length > 0 ? [this.variables[0].name.substr(1)] : [];
         return element.pivotUI(
```

Comment:
Why do you make an array from defaultVariable, if - as far as I can see - you only use one value?

Reply:
Thanks for the clarification!